src/checkstyle/checkstyle.xml

```diff
@@ -11,10 +11,6 @@
         <property name="headerFile" value="${checkstyle.header.file}"/>
         <property name="fileExtensions" value="java"/>
     </module>
-    <module name="SuppressionFilter">
-        <property name="file" value="src/checkstyle/checkstyle-suppressions.xml"/>
```

Comment:
I think we need to consider to use here this: https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html#suppressionsLocation. I mean `checkstyle.suppressions.location` property.

And suggest it for the `spring-cloud-build` instead of that wrong fix...

Reply:
Sorry could you clarify how to use the `checkstyle.suppressions.location` property? Do we no longer include it in the pom?

Reply:
Got it. I tried briefly and it seemed that including it in the pom alone was sufficient, however for the sake of consistency I will include it here as well. It would be needed if the checkstyle was ever run outside the context of maven.

Reply:
I discussed with @dmitry-s and decided to ultimately remove the placeholder and keep the suppression in the pom file. This is because the checkstyle.xml will eventually be removed so it is better to keep the configuration in the pom.

According to my research, defining suppression in the pom is [equivalent to defining it in the checkstyle](https://stackoverflow.com/questions/43871601/defining-suppressions-filter-in-checkstyle-xml-vs-pom-xml-maven-checkstyle-plu).

Reply:
So, we should consider to PR Spring Cloud Build to fix the parent `checkstyle.xml` similar way.
Although I'm not sure that we should specify `checkstyle-suppressions.xml` in their POM...

Reply:
@artembilan Do we need to revert https://github.com/spring-cloud/spring-cloud-build/pull/79?

Reply:
Yes, please.

Thanks

Reply:
@dmitry-s PTAL.